{
	"bugzilla": {
		"description": "CVE-2020-25813 mediawiki: Special:UserRights exposes the existence of hidden users",
		"id": "1903764",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1903764"
	},
	"csaw": false,
	"cvss3": {
		"cvss3_base_score": "5.3",
		"cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N",
		"status": "draft"
	},
	"cwe": "CWE-200",
	"details": [
		"In MediaWiki before 1.31.10 and 1.32.x through 1.34.x before 1.34.4, Special:UserRights exposes the existence of hidden users."
	],
	"name": "CVE-2020-25813",
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:openshift:3.11",
			"fix_state": "Will not fix",
			"package_name": "mediawiki",
			"product_name": "Red Hat OpenShift Container Platform 3.11"
		},
		{
			"cpe": "cpe:/a:redhat:openshift:4",
			"fix_state": "Out of support scope",
			"package_name": "mediawiki",
			"product_name": "Red Hat OpenShift Container Platform 4"
		}
	],
	"public_date": "2020-09-27T00:00:00Z",
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2020-25813\nhttps://nvd.nist.gov/vuln/detail/CVE-2020-25813\nhttps://phabricator.wikimedia.org/T232568"
	],
	"statement": "The mediawiki package was removed from OpenShift Container Platform in version 4.3.",
	"threat_severity": "Moderate",
	"upstream_fix": "mediawiki 1.31.10, mediawiki 1.34.4"
}
